File: eventmanagement/src/main/java/ch/fhnw/edu/wodss/eventmgmt/models/EventAttendeeFlat.java
package ch.fhnw.edu.wodss.eventmgmt.models;

import java.io.Serializable;

/**
 * "Slim version" of the event-attendees enriched by some user-properties
 *
 * Created by lukasschonbachler on 27.03.17.
 */
public class EventAttendeeFlat implements Serializable {

    /**
     * The firstname of the event-attendee
     */
    public String firstName;

    /**
     * The lastname of the event-attendee
     */
    public String lastName;

    /**
     * Indicates whether or not the attendee is internal (i.e. has an aai-login)
     */
    public boolean internal;

    /**
     * The selected food-type of the attendee (vegi, normal or none)
     */
    public String foodType;

    /**
     * Indicates whether or not the attendee selected a drink.
     */
    public boolean drink;
}

File: ionic/src/models/EventAttendeeFlat.ts
import {FoodType} from "./FoodType";

export class EventAttendeeFlat {
  /**
   * First name of the attendee
   */
  firstName: string;

  /**
   * Last name of the attendee
   */
  lastName: string;

  /**
   * Indicate if this is an internal or external user (Switch AAI is external)
   */
  internal: boolean;

  /**
   * What kind of food did the attendee order?
   */
  foodType: FoodType;

  /**
   * Did the attendee order a drink?
   */
  drink: boolean;
}
